.travis.yml

```diff
@@ -4,3 +4,5 @@ node_js:
   - "12"
   - "13"
```

Comment:
Let's drop these since they're pretty old. 14 is currently the oldest maintained version. https://github.com/nodejs/release#release-schedule